src/main/java/sg/dex/starfish/Resolver.java

```diff
@@ -38,6 +38,7 @@
      *
      * @param did DID to register
      * @param ddoString DDO to associate with the given DID
+     * @return Result
```

Comment:
What does the return value mean? Needs to be documented. I think I prefer throwing an exception if registration does not work?

Reply:
Done. Documented. But Interfaces will be changed again so it will be needed to review again anyway.